aip/0160.md

```diff
@@ -117,6 +129,16 @@ traversal through a message, map, or struct.
 | `a.b > 42`      | True if `a` has a numeric `b` field that is above 42. |
 | `a.b.c = "foo"` | True if `a.b` has a string `c` field that is "foo".   |
 
+Traversal **must** be written using the field names from the resource. If a
```

Comment:
Field names as specified in the original proto, presumably.

For field masks, the JSON representation performs camel casing... presumably there's nothing equivalent here, at least not yet? (I can see how that state of affairs could lead to some odd query strings - if you specify both a filter and a field mask in a query string, the two could specify the same field in different ways.)

Reply:
I actually think our filtering implementations accept both?

Reply:
They're supposed to accept both forms